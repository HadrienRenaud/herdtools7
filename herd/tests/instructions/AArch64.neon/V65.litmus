AArch64 V65
(* Tests LDAP1/STL1 single structure with 64 bit lanes *)
{
uint64_t x=1; uint64_t y;
0:X0=x; 0:X1=y;
}
 P0                   ;
 LDAP1 {V0.D}[1],[X0] ;
 STL1 {V0.D}[1],[X1]  ;
forall(y=1)
