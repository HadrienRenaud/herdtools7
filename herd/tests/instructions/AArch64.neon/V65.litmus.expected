Test V65 Required
States 1
[y]=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall ([y]=1)
Observation V65 Always 1 0
Hash=7b6eb8f377ec0aaadc98fc925703166f

